src/main/groovy/org/gradle/android/workarounds/RoomSchemaLocationWorkaround.groovy

```diff
@@ -179,6 +225,24 @@ class RoomSchemaLocationWorkaround implements Workaround {
         }
     }
 
+    private static void errorIfAnnotationProcessorSet(Set<String> options) {
```

Comment:
maybe rename this to `errorIfRoomSchemaSet`?

I say this because it is possible to use the room annotation processor correctly without the schema migrations and testing.

Reply:
renamed to `errorIfRoomSchemaAnnotationArgumentSet`